types/react-syntax-highlighter/index.d.ts

```diff
@@ -9,17 +9,39 @@
 
 type lineTagPropsFunction = (lineNumber: number) => React.HTMLProps<HTMLElement>;
 
+interface Style {
+    [selector: string]: {
+        [property: string]: string;
+    };
+}
+
+interface RendererParams {
+    rows: any;
+    stylesheet?: any;
+    useInlineStyles?: boolean;
+}
+
+type Renderer = (params: RendererParams) => React.ReactNode;
+
 declare module "react-syntax-highlighter" {
```

Comment:
If this is a 13.0 definition (like the presence of `showInlineLineNumbers` implies), this is missing the `createElement` export